wordpress-mobile/WordPress-iOS
# Aztec options picker popover DarkMode

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/12320#issuecomment-533688771

![aztec-picker](https://user-images.githubusercontent.com/9772967/66110844-e5e8c280-e5c7-11e9-9ac9-c186dd755100.png)


Related Aztec PR: https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1216

To test:
- Run the project on iOS 13 using DarkMode.
- Open Aztec.
- Select a Header and a List option on the toolbar.
- Check that the popover has a dark background.

Update release notes:
